packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.test.tsx

```diff
@@ -111,6 +124,344 @@ describe('routes/Product/SubscriptionUpgrade', () => {
+          'I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>$5.00 yearly</strong>, according to payment terms, until I cancel my subscription.';
+
+        const props = {
+          ...baseProps,
+          selectedPlan: plan,
+        };
+        const testRenderer = TestRenderer.create(
+          <SubscriptionUpgrade {...props} />
+        );
+        const testInstance = testRenderer.root;
+        const legalCheckbox = testInstance.findByProps({ id: expectedMsgId });
+
+        expect(legalCheckbox.props.$amount).toBe('5.00');
+        expect(legalCheckbox.props.$intervalCount).toBe(1);
+        expect(legalCheckbox.props.children.props.children).toBe(expectedMsg);
+      });
+
+      it('renders Localized for years plan with correct props and displays correct default string', async () => {
+        const plan_id = 'plan_6years';
+        const plan = findMockPlan(plan_id);
+        const expectedMsgId = 'sub-update-confirm-year';
+        const expectedMsg =
+          'I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>$5.00 every 6 years</strong>, according to payment terms, until I cancel my subscription.';
+
+        const props = {
+          ...baseProps,
+          selectedPlan: plan,
+        };
+        const testRenderer = TestRenderer.create(
+          <SubscriptionUpgrade {...props} />
+        );
+        const testInstance = testRenderer.root;
+        const legalCheckbox = testInstance.findByProps({ id: expectedMsgId });
+
+        expect(legalCheckbox.props.$amount).toBe('5.00');
+        expect(legalCheckbox.props.$intervalCount).toBe(6);
+        expect(legalCheckbox.props.children.props.children).toBe(expectedMsg);
+      });
+    });
+
+    describe('Fluent Localized Text', () => {
+      let bundle: FluentBundle;
+      const args = {
+        amount: '5.00',
+      };
+
+      beforeEach(async () => {
+        const filepath = path.join(
+          __dirname,
+          '..',
+          '..',
+          '..',
+          '..',
+          'public',
+          'locales',
+          'en-US',
+          'main.ftl'
+        );
+        const enUS = (await fs.readFileSync(filepath)).toString();
+        bundle = new FluentBundle('en-US');
+        bundle.addMessages(enUS);
+      });
+
+      describe('when the localized id is sub-update-confirm-day', () => {
+        const msgID = 'sub-update-confirm-day';
+
+        it('returns the correct string for an interval count of 1', () => {
+          const expected =
+            'I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>$5.00 daily</strong>, according to payment terms, until I cancel my subscription.';
+
+          const msg = bundle.getMessage(msgID);
+          const actual = bundle.format(msg, {
+            ...args,
+            intervalCount: 1,
+          });
+
+          expect(actual.replace(/(\u2068|\u2069)/gu, '')).toEqual(expected);
```

Comment:
This might just be how fluent works but why do these values need to be replaced?  Does `bundle.format` have options to omit it?

Reply:
From all of my testing, I couldn't find a way to omit these "hidden" characters. They kept getting added around each of the variables that are displayed.